openldap/2.6/debian-11/rootfs/opt/bitnami/scripts/libopenldap.sh

```diff
@@ -1037,8 +1037,16 @@ objectClass: olcOverlayConfig
 objectClass: olcSyncProvConfig
 olcOverlay: syncprov
 olcSpCheckpoint: $LDAP_SYNCPROV_CHECKPOINT
+EOF
+    if $LDAP_ENABLE_ACCESSLOG; then
+    cat >> "${LDAP_SHARE_DIR}/syncprov_create_overlay_configuration.ldif" << EOF
+olcSpSessionLogSource: $LDAP_SYNCPROV_SESSIONLOG
```

Comment:
Read the syncprov documentation what needs to be in here. Also pretty sure using LDAP_ENABLE_ACCESSLOG in a raw `if` isn't guaranteed to work (think about what happens when `LDAP_ENABLE_ACCESSLOG=yes`).

Reply:
Also first timers might never care about carrying an accesslog DB, so `no` sounds like a better default (especially if syncprov isn't enable either).